lib/main_common.pm

```diff
@@ -2409,6 +2409,7 @@ sub load_installation_validation_tests {
     # - autoyast/verify_ext4: validate installation using autoyast_ext4 profile
     # - console/verify_no_separate_home.pm: validate if separate /home partition disabled
     # - console/verify_separate_home.pm: validate if separate /home partition enabled
+    # - console/validate_autoyast: validate autoyast installation
```

Comment:
I believe name is too generic, it's not only to validate autoyast installation, but default installation. Putting `smoke` in the module name already helps.

Reply:
i have changed it to autoyast_smoke